tests/test_runner/test_discover_runner.py

```diff
@@ -332,6 +333,34 @@ def test_timings_captured(self):
         self.assertTrue(isinstance(runner.time_keeper, TimeKeeper))
         self.assertIn('test', stderr.getvalue())
 
+    def test_log(self):
+        cases = [
+            (0, None, False),
+            (0, logging.DEBUG, False),
+            (0, logging.INFO, False),
+            (0, logging.WARNING, False),
+            (1, None, True),
+            (1, logging.DEBUG, False),
+            (1, logging.INFO, True),
+            (1, logging.WARNING, True),
+            (2, None, True),
+            (2, logging.DEBUG, True),
+            (2, logging.INFO, True),
+            (2, logging.WARNING, True),
+            (3, None, True),
+            (3, logging.DEBUG, True),
+            (3, logging.INFO, True),
+            (3, logging.WARNING, True)
```

Comment:
Should there be a test for custom logging level?

Reply:
I added tests for custom levels.